test/ux-capture.test.js

```diff
@@ -0,0 +1,25 @@
+(function(window) {
+  //
+  // Run in either Mocha, Karma or Browser environments
```

Comment:
do we need to run this in a browser? I think we can avoid the boilerplate needed to support environments other than Jest